diff --git a/dnf/rpm/__init__.py b/dnf/rpm/__init__.py
index 488f5360b..1d50e6a0e 100644
--- a/dnf/rpm/__init__.py
+++ b/dnf/rpm/__init__.py
@@ -45,6 +45,17 @@ def detect_releasever(installroot):
             msg = 'Error: rpmdb failed to list provides. Try: rpm --rebuilddb'
             raise dnf.exceptions.Error(msg)
         releasever = hdr['version']
+        try:
+            off = hdr[rpm.RPMTAG_PROVIDENAME].index(distroverpkg)
+            flag = hdr[rpm.RPMTAG_PROVIDEFLAGS][off]
+            ver = hdr[rpm.RPMTAG_PROVIDEVERSION][off]
+            if flag == rpm.RPMSENSE_EQUAL and ver:
+                if hdr['name'] != distroverpkg:
+                    # override the package version
+                    releasever = ver
+        except (ValueError, KeyError, IndexError):
+            pass
+
         if is_py3bytes(releasever):
             releasever = str(releasever, "utf-8")
         return releasever
